dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -841,6 +841,10 @@ def disabled?(selector)
   @browser.execute_script("__TestInterface.setSpeedSliderValue(0.8)")
```

Comment:
Do you know where this was getting used in the past?

Reply:
As you mentioned- I couldn't find any current usages but I know it's defined in loadArtist.js. I tried to mimic that design with @maureensturgeon 's help, but we loadJava so differently that it just seemed to make a lot more sense/be a lot cleaner to have it directly in Neighborhood.js